cvat-ui/src/components/tasks-page/task-item.tsx

```diff
@@ -239,6 +239,10 @@ class TaskItemComponent extends React.PureComponent<TaskItemProps & RouteCompone
             history.push(`/tasks/${taskInstance.id}/analytics`);
```

Comment:
It would be nice to remember unfolded normal jobs having consensus jobs when opening these jobs from the task page. It gets a little bit annoying to open these lists each time you're returning back from a consensus job review and want to open another one.